index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PORTFOLIO</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  </head>
+  <body>
+    <header class="hero">
+      <nav>
+        <a href="#" class="logo">Welcome</a>
+        <ul class = "nav-list">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <label for="check" class="hambuger-btn">
+          <i class = "fas fa-bars"></i>
+        </label>
+      </nav>
+    </header>
+    <section>
+      <div class="content">
+        <h1>Hey there. I'm Esther</h1>
+        <h2>I'm a software developer</h2>
+        <P class = "supporting-text">I can help you build a feature, product or a website.
+        Look through some of my work and experience!
+        If you like what you see and have a project you need coded, dont
+          hesitate to contact me.
+        </p>
+      </div>
+      <section>
+      <ul class="socialmedia">
+        <li>
+          <a href="https://twitter.com/EstherMawioo">
+              <img src="./images/socialmedia icons/twitter.png" alt="twitter icon">
+          </a>
+        </li>
+        <li>
```

Comment:
- [ ] As requested by the previous reviewer, let's try to stick to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) 🎨  as much as possible, this includes background images. Remember that having a good and professional-looking portfolio will make you stand out from other developers, so let's try and follow the template as closely as possible! 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/198086909-d7c74e44-dc8d-45af-b64a-1204331641ff.png) | ![image](https://user-images.githubusercontent.com/77069011/198086970-870bbf13-fe15-4b7d-b87e-fb679a1bd8ac.png) |

- [ ] **[OPTIONAL]:** It would be a great idea if you can fix the spacing between the toolbar and the hero section too!. 🔧